src/main/mainwindow.cpp

```diff
@@ -1131,7 +1131,7 @@ inline void MainWindow::slotShortcutHorizonzalSplit()
     // 判读数量是否允许分屏
     if (Service::instance()->isCountEnable()) {
         TermWidgetPage *page = currentPage();
-        if (page && CanSplit(page->currentTerminal(), Qt::Vertical)) {
+        if (page && TermWidget::canSplit(page->currentTerminal(), Qt::Vertical)) {
```

Comment:
I meant "convert canSplit to object function". So that, the way of calling it becomes
```C++
page->currentTerminal()->canSplit(Qt::Vertical)
```